ci/build_wheel.sh

```
@@ -42,33 +42,26 @@ sccache --stop-server 2>/dev/null || true
-    RAPIDS_PIP_WHEEL_ARGS+=(--build-constraint="${PIP_CONSTRAINT}")
-fi
-
-# unset PIP_CONSTRAINT (set by rapids-init-pip)... it doesn't affect builds as of pip 25.3, and
-# results in an error from 'pip wheel' when set and --build-constraint is also passed
+# `build` receives the same generated constraints explicitly. Unset the
+# environment variable so it does not constrain the frontend installation.
 unset PIP_CONSTRAINT
 
-rapids-telemetry-record build-${package_name}.log rapids-pip-retry wheel \
-    "${RAPIDS_PIP_WHEEL_ARGS[@]}" \
+rapids-telemetry-record build-${package_name}.log python -m build \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)